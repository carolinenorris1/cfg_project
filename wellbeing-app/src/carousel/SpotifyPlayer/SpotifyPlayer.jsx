import React from "react";

function SpotifyPlayer() {
  return (
    <iframe
      src="https://open.spotify.com/embed/playlist/5qQBuK3Q0iYlGmrjJCy6Ri?utm_source=generator"
      width="100%"
      height="675"
      frameBorder="0"
      allowfullscreen=""
      allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"
      loading="lazy"
    ></iframe>
  );
}

export default SpotifyPlayer;
