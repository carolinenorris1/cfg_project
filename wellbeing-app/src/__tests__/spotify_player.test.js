import React from "react";
import renderer from "react-test-renderer";

import SpotifyPlayer from "../carousel/SpotifyPlayer/SpotifyPlayer";

it("renders correctly when user views the spotify player on the carousel", () => {
  const tree = renderer.create(<SpotifyPlayer />).toJSON();
  expect(tree).toMatchSnapshot();
});
