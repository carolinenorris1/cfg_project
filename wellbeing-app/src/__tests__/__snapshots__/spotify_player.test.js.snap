// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when user views the spotify player on the carousel 1`] = `
<iframe
  allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"
  allowfullscreen=""
  frameBorder="0"
  height="675"
  loading="lazy"
  src="https://open.spotify.com/embed/playlist/5qQBuK3Q0iYlGmrjJCy6Ri?utm_source=generator"
  width="100%"
/>
`;
